MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -5734,19 +5734,19 @@ public int findAtBPartsAvailabilityLevel(IAcquisitionWork acquisition, StringBui
          * If the unit is not assigned to a contract, use the least restrictive active
          * contract. Don't restrict parts availability by contract if it has not started.
          */
-        if (hasActiveContract() && (contract == null)) {
+        if (hasActiveContract()) {
             for (Contract c : getActiveContracts()) {
-                if (c instanceof AtBContract && 
-                        c.getStartDate().isBefore(currentDay) && 
-                        (contract == null || 
+                if ((c instanceof AtBContract) && 
+                        (c.getStartDate().isBefore(currentDay) || c.getStartDate().equals(currentDay)) && 
+                        ((contract == null) || 
```

Comment:
These three are all cancelled out by the getActiveContracts, which only returns those whose date is active.